pepper-apis/src/main/java/org/broadinstitute/ddp/event/pubsubtask/api/PubSubTaskReceiver.java

```diff
@@ -31,6 +33,10 @@
 
     private static final Logger LOG = getLogger(PubSubTaskReceiver.class);
 
+    private static final int RETRY_MESSAGE_MAX_COUNT = 5;
```

Comment:
Pubsub supports limiting retries by tweaking the settings of the subscription. Have we considered that?

Reply:
I tried to find some standard approach which could be good but without any luck..
What happens when limit (configured in subscription) is gained?
Some kind of exception thrown?

Reply:
Ah, I understood: it  is published to a special topic...